benchmark/feldera-sql/benchmarks/nexmark/queries/q15.sql

```diff
@@ -0,0 +1,17 @@
+CREATE VIEW Q15 AS
```

Comment:
We moved q15 to disabled-queries because it broke the CI tests by hanging. If this fixes that then please move it back to this directory but in either case please don't have a q15.sql in both directories.